tests/test_compat_matrix.py

```diff
@@ -46,3 +46,5 @@ def test_12(self):
     def test_13(self):
         assert osadl_matrix.get_compatibility("Unfairy-license", "EPL-2.0")  == osadl_matrix.OSADLCompatibility.UNDEF
 
+    def test_supported_licenes(self):
+        assert len(osadl_matrix.supported_licenses()) > 20
```

Comment:
This test is not enough - pls test for selected items being part of the result list